src/app/features/home-feature/components/templates/login-template/login-template.component.html

```diff
@@ -1,12 +1,15 @@
-<div class="login-template-container">
+<form class="login-template-container" [formGroup]="loginForm" (ngSubmit)="onSubmit()">
   <sk-card class="card">
     <h1 class="title">Logg inn</h1>
-    <sk-input-email class="email" [placeholder]="'E-post'" (valueChange)="onEmailChanged($event)"></sk-input-email>
-    <sk-input-password class="password" [placeholder]="'Passord'" (valueChange)="onPasswordChanged($event)"></sk-input-password>
-    <sk-button class="login-button" [variant]="'primary'" (clicked)="onLogin()"> Logg inn </sk-button>
+    <sk-input-text class="email" [type]="'email'" [placeholder]="'E-post'" formControlName="email"></sk-input-text>
+    <sk-input-text class="password" [type]="'password'" [placeholder]="'Passord'" formControlName="password"></sk-input-text>
+    <div class="errors" *ngIf="stateError || (attemptedSubmit && loginForm.invalid && (loginForm.dirty || loginForm.touched))">
+      <span class="error-message">Ugyldig brukernavn eller passord</span>
+    </div>
```

Comment:
This is the correct placement, however because the login template is placed in a vertically centered container, the fields move away from the center when the template grows, i.e. an error is shown.

This is a hard problem to solve, but because we know the size of the element and the length of the text, we can cheat. A solution can be to add a few lines of whitespace:

```suggestion
    <div class="no-errors" *ngIf="!stateError && !(attemptedSubmit && loginForm.invalid && (loginForm.dirty || loginForm.touched))">
      <br />
      <br />
    </div>
    <div class="errors" *ngIf="stateError || (attemptedSubmit && loginForm.invalid && (loginForm.dirty || loginForm.touched))">
      <span class="error-message">Ugyldig brukernavn eller passord</span>
    </div>
```